holidays/countries/india.py

```
@@ -492,47 +515,188 @@ def _populate_subdiv_br_public_holidays(self):
 
     # Chandigarh.
     def _populate_subdiv_ch_public_holidays(self):
-        pass
+        # Guru Gobind Singh's Birthday.
+        self._add_guru_gobind_singh_jayanti(tr("Guru Gobind Singh's Jayanti"))
+        # Maharshi Valmiki's Birthday.
+        self._add_maharshi_valmiki_jayanti(tr("Maharshi Valmiki's Jayanti"))
+
+    def _populate_subdiv_ch_optional_holidays(self):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_populate_subdiv_ch_optional_holidays`

Add return type annotation: `None`

(ANN202)